apis/python/src/tiledbsoma/util_arrow.py

```diff
@@ -121,36 +121,31 @@ def get_arrow_schema_from_tiledb_uri(
     return pa.schema(arrow_schema_dict)
 
 
-def ascii_to_unicode_pyarrow_readback(record_batch: pa.RecordBatch) -> pa.RecordBatch:
+def ascii_to_unicode_pyarrow_readback(table: pa.Table) -> pa.Table:
     """
     Implements the 'decode on read' part of our ASCII/Unicode logic
     """
     # TODO: COMMENT/LINK HEAVILY
-    names = [ofield.name for ofield in record_batch.schema]
+    names = [ofield.name for ofield in table.schema]
     new_fields = []
     for name in names:
-        old_field = record_batch[name]
-        if isinstance(old_field, pa.LargeBinaryArray):
+        old_field = table[name]
```

Comment:
Just confirming this is known to work with chunked arrays (which can happen for very large query results, ie, column size > 2GB)

Reply:
@bkmartinjr to be clear the _column_ is not the `pa.LargeBinaryArray` here but rather the _cell value_

Reply:
also reminding this will go away entirely on #359

Reply:
(that's how Unicode encoded as bytes reads back in Arrow unless we actively decode it -- hence #359 which will remove that need)

Reply:
I believe you misunderstood my question.  A column in a table is ChunkedArray.  If there is more than one chunk in the table column, is it correctly handled?  

I believe I have convinced myself it is fine, as `pa.array` on line 134 will concatenate all of the chunks into a single `Array`.